packages/rke2-calico/generated-changes/patch/templates/crs/custom-resources.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/templates/crs/custom-resources.yaml
++++ charts/templates/crs/custom-resources.yaml
+@@ -6,6 +6,12 @@
+ {{ $secrets = append $secrets $item }}
+ {{ end }}
+ {{ $_ := set $installSpec "imagePullSecrets" $secrets }}
++{{ $networkSpec := get $installSpec "calicoNetwork" }}
++{{ $ipPools := get $networkSpec "ipPools" }}
++{{ $defaultipPools := first $ipPools }}
++{{ $defaultCIDR := get $defaultipPools "cidr" }}
```

Comment:
Is the default here the same as RKE2 - 10.42.0.0/16?

Reply:
yes ==> https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-calico/generated-changes/patch/values.yaml.patch#L12

Reply:
Do you know a way to reach cidr in one line?